go/controller/autoscaler/autoscaler.go

```diff
@@ -81,10 +81,9 @@ func (j jobs) Less(a int, b int) bool {
 	if scoreA == scoreB {
 		resA := j[a].Config.Spec.Trainer.Resources
 		resB := j[b].Config.Spec.Trainer.Resources
-		// FIXME: use Quantity type, should refine these code.
 		resALimitsGPU := resA.Limits[paddlejob.GPUResourceName]
 		resBLimitsGPU := resB.Limits[paddlejob.GPUResourceName]
-		if resALimitsGPU.Cmp(resALimitsGPU) == 0 {
+		if resALimitsGPU.Cmp(resBLimitsGPU) == 0 {
```

Comment:
It's hard to find bug without test :)

Reply:
Will add later.

Reply:
@typhoonzero Already have test here, I found this line after fixing the build of the test.